quartz/styles/base.scss

```diff
@@ -184,7 +184,7 @@ a {
       box-sizing: border-box;
       padding: $topSpacing 2rem 2rem 2rem;
       display: flex;
-      height: 100vh;
+      min-height: 100vh;
```

Comment:
hmm, this would make side panel scrollable past vh


https://github.com/user-attachments/assets/b0f3962c-2045-4d85-b88e-104a68f2d18b

in this case I have sidebar set to `display: grid`. Not sure if this affects anything.

Reply:
I have a suspicion that it is caused by not setting a height. I'll investigate later today.

Reply:
no rush. Thanks for the prompt update.